app/src/main/res/drawable/ic_quest_collection_times.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="utf-8"?>
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
AFAIK there is already an icon for this: `ic_quest_mail` or something like that...

Reply:
Yes, there is. Please use that one.